### PR TITLE
update shacl shapes

### PR DESCRIPTION
**Pull request by Nora-Olivia-Ammann**

### CODE REVIEW

File: journeyStar_shacl.ttl [hunk 1]
```diff
@@ -148,6 +148,36 @@ js-shacl:JourneyShape a sh:NodeShape ;
         sh:node js-shacl:LocationShape ;
     ] .
 
+### Shape for Class js:Journey
```
Comment (Nora-Olivia-Ammann) [inline journeyStar_shacl.ttl:154]: I have created a shape for the superclass without cardinalities. This way we should only need to define the cardinalities here.

File: journeyStar_shacl.ttl [hunk 2]
```diff
@@ -145,7 +145,40 @@ js-shacl:JourneyShape a sh:NodeShape ;
     ] ,
     [
         sh:path js:transitThrough ;
-        sh:node js-shacl:LocationShape ;
+        sh:or ( [ sh:datatype xsd:string ]
```
Comment (Nora-Olivia-Ammann) [inline journeyStar_shacl.ttl:148]: I noticed that locations are inconsistent. Was this on purpose that some properties have location targets that were more restrictive? I now changed all to the less restrictive option.

Comment (SepidehAlassi) [reply journeyStar_shacl.ttl:148]: It does not need to be strictly of type Location, making it generic like you have done is fine.

File: journeyStar_shacl.ttl [hunk 3]
```diff
@@ -145,7 +145,40 @@ js-shacl:JourneyShape a sh:NodeShape ;
...
+    ] ,
+    [
+        sh:path js:endDate ;
+        sh:maxCount 1 ;
+    ] ,
+    [
+        sh:path js:destination ;
+        sh:disjoint js:origin ;
+        sh:severity sh:Violation ;
+        sh:message "A Journey is specified that the destination and origin are not the same location."@en;
```
Comment (SepidehAlassi) [inline journeyStar_shacl.ttl:181]: change the message to: `"A journey cannot have the same origin and destination. For round trips, the entire journey must be defined in stages!"`

**Review by SepidehAlassi — APPROVED** [review]

**Pull request merged**